cloud_functions/shared/task_utils.py

```
@@ -0,0 +1,56 @@
+
+    Returns:
+        str: The encrypted token in the format "iv:encrypted", where both the IV and the ciphertext are base64-encoded.
+    """
+    # Convert the plaintext token to bytes.
+    token_bytes = token.encode("utf-8")
+
+    # Convert the key to bytes if necessary.
+    if isinstance(key, str):
+        key_bytes = key.encode("utf-8")
+    else:
+        key_bytes = key
```

Ruff (0.8.2)

Use ternary operator `key_bytes = key.encode("utf-8") if isinstance(key, str) else key` instead of `if`-`else`-block

Replace `if`-`else`-block with `key_bytes = key.encode("utf-8") if isinstance(key, str) else key`

(SIM108)